### DISCUSSION TITLE
DInfo authoring tools

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1255] [General]
According to the documentation ot the DInfo LispUsers package:

> Graphs for the *Interlisp-D Reference Manual* and the *Xerox Quintus Prolog Manual* are available, as are tools for creating and editing new graphs.

Aside from TEdit what other tools for creating and editing new graphs, if any, does the documentation refer to?

**Comment by Anzus** [comment]
The Doctools directory on GitHub has the tools for making indexes (and documentation for use).  I haven’t been able to get them to build the graphs for the new IRM yet, but TBH I haven’t looked into why that is yet.

**Reply by pamoroso** [reply]
Thanks, I was just curious.
